### ISSUE TITLE
关于yolov3损失函数的接口

### ISSUE THREAD

**Issue by 670133189** [PaddlePaddle/Paddle#37026]
参考官方 yolov3 的实现【https://www.paddlepaddle.org.cn/tutorials/projectdetail/2412195#anchor-30】
里面使用的损失函数是`binary_cross_entropy_with_logits()`，即 **logits二值交叉熵损失值**
但根据官方的 yolo_loss 接口【https://www.paddlepaddle.org.cn/documentation/docs/zh/api/paddle/vision/ops/yolo_loss_cn.html#yolo-loss】
提示是 **sigmoid交叉熵损失**，即先将 x 值进行 sigmoid，再进行`binary_cross_entropy()`

与官方`paddle.vision.ops.yolo_loss()`结果对比：
使用`binary_cross_entropy()`，最终输出的 loss 等于官方`yolo_loss()`
使用`binary_cross_entropy_with_logits()`，结果为官方`yolo_loss()`的2倍

请问我的理解正确吗？
